### PR TITLE
feat: product configuration pages loading skeletons

### LINKED ISSUE #3213: [🕹️] Loading skeletons for Configuration pages
### Is your feature request related to a problem? Please describe.

We currently don't have a loading skeleton for the product config pages

### Describe the solution you'd like

Add one :)

### Task
1. Look at how we handled it on the People page with the tabs for Segments and Attributes. Use the same approach to not reload the tab bar for a better effect
2. For each of the subpages (general, look & feel, etc..) it's enough to reuse the same loading.tsx of a single card

Please familiarize yourself with best practices of loading skeletons in NextJS before working on this task 🤓 

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by PratikAwaik**
## What does this PR do?

Fixes #3213 

## How should this be tested?

- Navigate to `/product/general` by clicking on Configuration tab in the sidebar.
- Switch tabs to see the updated loading state for all tabs.

### Required

- [x] Filled out the "How to test" section in this PR
- [x] Read [How we Code at Formbricks](<[https://github.com/formbricks/formbricks/blob/main/CONTRIBUTING.md](https://formbricks.com/docs/contributing/how-we-code)>)
- [x] Self-reviewed my own code
- [x] Commented on my code in hard-to-understand bits
- [x] Ran `pnpm build`
- [x] Checked for warnings, there are none
- [x] Removed all `console.logs`
- [x] Merged the latest changes from main onto my branch with `git pull origin main`
- [x] My changes don't cause any responsiveness issues
- [x] First PR at Formbricks? [Please sign the CLA!](https://cla-assistant.io/formbricks/formbricks) Without it we wont be able to merge it 🙏

### Appreciated

- [ ] If a UI change was made: Added a screen recording or screenshots to this PR
- [x] Updated the Formbricks Docs if changes were necessary

## Release Notes

- **New Features**
	- Introduced a `ProductConfigNavigation` component for improved navigation across various loading screens.
	- Enhanced the `Loading` component with clearer status messages regarding app connections.
	- Added a new `SettingsCard` to display setup instructions for the Formbricks widget.

- **Bug Fixes**
	- Updated descriptions for clarity in several components, ensuring consistent punctuation.

- **Refactor**
	- Streamlined loading components by removing unnecessary navigation logic and integrating new components for better organization.
	- Improved modularity and readability of loading components by centralizing definitions and simplifying structure. 

- **Chores**
	- Removed tooltip functionality in the `EditLanguage` component to simplify UI.

### CODE REVIEW

File: apps/web/app/(app)/environments/[environmentId]/product/(setup)/website-connection/loading.tsx [hunk 1]
```diff
@@ -1,49 +1,41 @@
...
 
-import { BrushIcon, KeyIcon, LanguagesIcon, ListChecksIcon, TagIcon, UsersIcon } from "lucide-react";
-import { usePathname } from "next/navigation";
+import { ProductConfigNavigation } from "@/app/(app)/environments/[environmentId]/product/components/ProductConfigNavigation";
+import { SettingsCard } from "@/app/(app)/environments/[environmentId]/settings/components/SettingsCard";
 import { cn } from "@formbricks/lib/cn";
 import { PageContentWrapper } from "@formbricks/ui/components/PageContentWrapper";
 import { PageHeader } from "@formbricks/ui/components/PageHeader";
 
 const LoadingCard = ({ title, description, skeletonLines }) => {
```
Comment (jobenjada) [inline apps/web/app/(app)/environments/[environmentId]/product/(setup)/website-connection/loading.tsx:9]: We're having this LoadingCard in each loading.tsx of the settings -> Feels like it should be its own component.

File: apps/web/app/(app)/environments/[environmentId]/product/look/loading.tsx [hunk 2]
```diff
@@ -21,202 +20,140 @@ const placements = [
...
-                      <div className="h-3 w-3 rounded-full bg-emerald-500"></div>
-                    </div>
-                    <div className="ml-4 flex w-full justify-between font-mono text-sm text-slate-400">
-                      <p>Preview</p>
-
-                      <div className="flex items-center pr-6">Restart</div>
-                    </div>
+          <div className="relative flex w-1/2 flex-row items-center justify-center rounded-lg bg-slate-100 pt-4">
+            <div className="relative mb-3 flex h-fit w-5/6 items-center justify-center rounded-lg border border-slate-300 bg-slate-200">
+              <div className="flex h-[95] max-h-[90%] w-4/6 flex-1 flex-col">
```
[the remark this answers is not in the document]
Comment (jobenjada) [reply apps/web/app/(app)/environments/[environmentId]/product/look/loading.tsx:81]: @PratikAwaik pls solve

**Review by jobenjada — CHANGES_REQUESTED** [review]

**Comment by jobenjada** [comment]
happy to add +300 points to the issue if you can check the organization settings as well as the account settings for their implementation of loading skeletons and unify it. They all also have their own LoadingCard component which doesnt make sense imo

**Comment by PratikAwaik** [comment]
@jobenjada fixed PR comments as well as added unified loading in account and organization settings page

**Review by jobenjada — APPROVED** [review]

**Pull request merged**